dev/tests/integration/testsuite/Magento/Framework/View/Layout/Reader/_files/_layout_update_reference.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:noNamespaceSchemaLocation="../../../../../../../../../../lib/internal/Magento/Framework/View/Layout/etc/page_configuration.xsd">
+    <body>
+        <block class="Dummy\Class" name="test.block"/>
+        <referenceBlock name="test.block">
+            <arguments>
+                <argument name="test_arg" xsi:type="string">test-argument-value</argument>
+            </arguments>
+            <action method="setTestAction">
```

Comment:
Please remove action node from layout. It's deprecated. see: [Layout Test](https://github.com/magento/magento2/blob/develop/dev/tests/static/testsuite/Magento/Test/Legacy/LayoutTest.php#L201)